Tiltfile

```diff
@@ -13,7 +13,8 @@ load("ext://uibutton", "cmd_button", "location", "text_input")
 version_settings(True, ">=0.22.2")
 
 settings = {
-    "enable_providers": ["docker"],
+    "enable_providers": [],
```

Comment:
This change also disables deploying docker by default to allow a completely empty provider list. Users would now have to add docker to their tilt-settings explicitly.